subprojects/core-api/src/main/java/org/gradle/api/provider/ListProperty.java

```diff
@@ -30,4 +32,27 @@
  */
 @Incubating
 public interface ListProperty<T> extends PropertyState<List<T>> {
+    /**
+     * Adds an element to the property.
+     *
+     * @param element The element, can be null.
+     * @since 4.4
+     */
+    void add(@Nullable T element);
```

Comment:
why would you want to add a null element?

Reply:
Don't know :-( I will remove it. Should I add a precondition?

Reply:
Yes, I think that makes sense